packages/manager/apps/public-cloud/src/components/walkMe/templates/nextButton.html

```diff
@@ -0,0 +1,3 @@
+<button class="mr-1 oui-button oui-button_icon-alone {{ visibility }}" data-role="next">
```

Comment:
is this supposed to be `ARIA` ?

because `role="end"` doesn't exists: https://www.w3.org/TR/wai-aria-1.1/#end

:)

Reply:
This is relative to the used library http://bootstraptour.com/api/